### PR TITLE
GIX-1745: Neuron Settings Feature Flags flips between old and new UI

### PR DESCRIPTION
**Pull request by lmuntaner**
# Motivation

We are shuffling data in the neuron details page.

In this PR: The feature flag flips between the old and new UI.

# Changes

* Add the old cards of NnsNeuronDetails inside the "else" statement of the ENABLE_NEURON_SETTINGS
* Add the old cards of SnsNeuronDetails inside the "else" statement of the ENABLE_NEURON_SETTINGS
* Increase spacing between sections to adapt to old design. There is a task to fix that onece we are fully in the new design.

# Tests

* Test new sections are rendered in NnsNeuronDetails and SnsNeuronDetails. Add getters to the POs.

# Todos

- [x] Add entry to changelog (if necessary).

### CODE REVIEW

**Comment by lmuntaner** [comment]
NNS Neuron detail page:

Old:
nns-details

New:
nns-settings

SNS Neuron detail page:

Old:
sns-details

New:
sns-settings

**Comment by lmuntaner** [comment]
@dskloetd @mstrasinskis please review

File: frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts [hunk 1]
```diff
@@ -52,91 +53,127 @@ describe("NeuronDetail", () => {
...
 
-  const testTitle = async ({
-    intersecting,
-    text,
-  }: {
-    intersecting: boolean;
-    text: string;
-  }) => {
-    const { getByTestId } = render(NnsNeuronDetail, props);
+  describe("when old neuron details page", () => {
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts:82]: Test that the new section is not rendered?
(Sorry if you've already done that. GitHub misaligns the old code and the new code which makes it impossible to review.)

File: frontend/src/tests/lib/pages/SnsNeuronDetail.spec.ts [hunk 2]
```diff
@@ -132,6 +167,8 @@ describe("SnsNeuronDetail", () => {
         id: [validNeuronId],
         cached_neuron_stake_e8s: numberToE8s(neuronStake),
       });
+
+      overrideFeatureFlagsStore.setFlag("ENABLE_NEURON_SETTINGS", false);
```
Comment (dskloetd) [inline frontend/src/tests/lib/pages/SnsNeuronDetail.spec.ts:171]: Test that the new section is not rendered?

File: frontend/src/lib/pages/NnsNeuronDetail.svelte [hunk 3]
```diff
@@ -192,6 +194,6 @@
   .section-wrapper {
     display: flex;
     flex-direction: column;
-    gap: var(--padding-3x);
+    gap: var(--padding-4x);
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/lib/pages/NnsNeuronDetail.svelte:197]: Double negation, hehe. It's hard to follow the question.

This does NOT affect the layout if feature flag is not enabled.

File: frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts [hunk 4]
```diff
@@ -52,91 +53,127 @@ describe("NeuronDetail", () => {
...
 
-    await waitFor(() => {
-      const neuronIdElement = getByTestId("neuron-id");
-      expect(neuronIdElement).not.toBeNull();
-      expect(neuronIdElement.textContent).toEqual(`${neuronId}`);
-      expect(neuronIdElement.textContent).not.toEqual(`${mockNeuron.neuronId}`);
+      expect(await po.getVotingPowerSectionPo().isPresent()).toBe(true);
+      expect(await po.getMaturitySectionPo().isPresent()).toBe(true);
+      expect(await po.getAdvancedSectionPo().isPresent()).toBe(true);
     });
```
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts:79]: Sure, I can add it.

File: frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts:82]: No, I didn't add this one. I'll add it.

File: frontend/src/tests/lib/pages/SnsNeuronDetail.spec.ts (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/SnsNeuronDetail.spec.ts:171]: sure, good point

File: frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts (hunk 4 above)
[the remark this answers is not in the document]
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts:87]: done

File: frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts (hunk 1 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/NnsNeuronDetail.spec.ts:90]: done

File: frontend/src/tests/lib/pages/SnsNeuronDetail.spec.ts (hunk 2 above)
Comment (lmuntaner) [reply frontend/src/tests/lib/pages/SnsNeuronDetail.spec.ts:171]: done

**Comment by lmuntaner** [comment]
@dskloetd ready for another review

**Review by dskloetd — APPROVED** [review]
